Sources/GRPCHTTP2Core/Server/GRPCServerStreamHandler.swift

```diff
@@ -35,6 +35,7 @@ final class GRPCServerStreamHandler: ChannelDuplexHandler {
   // if there are messages still not written into the channel because flush has
   // not been called, but the server sends back trailers).
   private var pendingTrailers: HTTP2Frame.FramePayload?
+  private var pendingTrailersPromise: EventLoopPromise<Void>?
```

Comment:
These always come together, can we store them as `(HTTP2Frame.FramePayload, EventLoopPromise<Void>?)?`?